rpc/documentation/serverchanges.md

```diff
@@ -69,9 +69,13 @@ A `sh` shell script is included to compile the Protocol Buffers descriptor.  It
 must be run from the `rpc` directory.
 
 ```bash
-$ sh regen.sh
+$ sh regen-docker.sh
 ```
 
+This runs `protoc` inside a`docker` container, which is optimal when the dependencies of 
```

Comment:
missing a space between "a" and `docker`

Reply:
thx!